docs/config-file/v2.rst

```diff
@@ -154,13 +154,19 @@ python.version
 
 The Python version (this depends on :ref:`config-file/v2:build.image`).
 
-:Type: ``number``
+:Type: ``string``
```

Comment:
We should have a note about the historic value here, at least.

Reply:
I have added a small note that this can be a number, but isn't recommended.